legal-api/src/legal_api/models/corp_type.py

```diff
@@ -20,7 +20,6 @@
 class CorpType(db.Model):  # pylint: disable=too-many-instance-attributes
     """This class manages the corp type."""
 
-    __versioned__ = {}
```

Comment:
Does `corp_type` have version table? I don't find it so I remove its old versioning.

Reply:
couldn't find it in DEV db on my side 
![image](https://github.com/user-attachments/assets/3e2aa98d-3820-4412-bfbc-a81f8dcff5eb)